### PR TITLE
doc: document "rgw bucket quota ttl" setting

### PR DESCRIPTION
**Pull request by ktdreyer**
Signed-off-by: Ken Dreyer <kdreyer@redhat.com>

### CODE REVIEW

File: doc/radosgw/config-ref.rst [hunk 1]
```diff
@@ -367,6 +367,16 @@ Ceph configuration file, the default value will be set automatically.
 :Type: Integer
 :Default: ``-1``
 
+``rgw bucket quota ttl``
+
+:Description: Time (seconds) for cached bucket stats to be cached within this
```
Comment (oritwas) [inline doc/radosgw/config-ref.rst]: I would remove the extra cached:
Time (seconds) for bucket stats to be cached

**Comment by amedeos** [comment]
Could you add the default value of 600 seconds on the documentation?

File: doc/radosgw/config-ref.rst (hunk 1 above)
Comment (ktdreyer) [inline doc/radosgw/config-ref.rst]: Thanks Orit, I've amended and re-pushed with this change.

**Comment by ktdreyer** [comment]
superceded by https://github.com/ceph/ceph/pull/13395

**Pull request closed without merging**